subsystems/LinearServo.java

```diff
@@ -0,0 +1,45 @@
+package frc.team670.mustanglib.subsystems;
+
+import edu.wpi.first.wpilibj.Servo;
+
+
+/**
+ * Represents a Linear Servo
+ * 
+ * @author AkshatAdsule
+ */
+public class LinearServo {
+    
+    private final Servo actuator;
+
+    /**
+     * Creates a new LinearServo
+     * @param channel the PWM port 
+     */
+    public LinearServo(int channel) {
+        actuator = new Servo(channel);
+        actuator.setBounds(2.0, 1.8, 1.5, 1.2, 1.0);
+    }
+
+    /**
+     * Extends the servo to its maximum length
+     */
+    public void extend() {
+        actuator.setSpeed(1);
```

Comment:
is 1 for full extend and 0 for full retract common to linear actuators or specific to this one? I also wonder why we call set() to move to a specific point (presumably including all the way in and all the way out?) and we call setSpeed() to move all the way in or out. Is there a reason?

Reply:
yes, it should be set. FIxed in [f148aca](https://github.com/HHS-Team670/MustangLib/pull/10/commits/f148acab8de64a42e776ea716547ad6a286e2e4f)